tests/set-annotation-complete-input/fn-config.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-func-config
+  annotations:
+    config.k8s.io/function: |
+      container:
+        image: gcr.io/kpt-fn/set-annotation:unstable
+    config.kubernetes.io/local-config: 'true'
+data:
+  annotations:
```

Comment:
I'd treat the one in examples as a simple use case and this one an advanced use case. Put this example in `tests` will make it hard to discover.
We should come up with a way to present both use cases to our users in `examples`, and both should be tested.

Reply:
Re-organized the example directory.